docs/pip.md

```diff
@@ -484,6 +485,50 @@ hermeto-output/deps/pip
 └── ...
 ```
 
+## Working with Rust-based dependencies
+
+Hermeto provides a way to prepare hermetic build of a Python package which depends on
+on Rust-based packages. This process cannot be fully automated on Hermeto's side, thus it would
+require minor intervention from users.
+
+Building such project requires that all build dependencies are present. To achieve this
+[pybuild-deps](https://pypi.org/project/pybuild-deps) could be used. The following command
+will take care of generating the list:
+
+```
+echo <package_name==package_version> | pybuild-deps compile /dev/stdin --generate-hashes -o requirements-build.txt
```

Comment:
Is this the recommended invocation? So one cannot specify the package spec as a string on the command line and instead it needs to be read from stdin? That doesn't sound very robust or convenient and we should improve that since now it falls into our maintenance domain.